paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/resources/RepositoryLoader.kt

```diff
@@ -0,0 +1,1175 @@
+    var parentStyle = parser.getAttributeValue(null, ATTR_PARENT)
+    if (parentStyle != null && parentStyle.isNotEmpty()) {
+      urlParser.parseResourceUrl(parentStyle)
+      parentStyle = urlParser.qualifiedName
+    }
+
+    val resourceValue = StyleResourceValueImpl(namespace, name, parentStyle, null)
+    resourceValue.namespaceResolver = namespaceResolver
+
+    forSubTags(
+      TAG_ITEM,
+      object : XmlTagVisitor {
+        override fun visitTag() {
+          val itemNamespaceResolver = parser.namespaceResolver
+          val itemName = parser.getAttributeValue(null, ATTR_NAME)
+          if (itemName != null) {
+            val text = textExtractor.extractText(parser, false)
+            val styleItem = StyleItemResourceValueImpl(
+              namespace,
+              itemName,
+              text,
+              configuration.repository.libraryName
+            )
+            styleItem.namespaceResolver = itemNamespaceResolver
+            resourceValue.addItem(styleItem)
+          }
+        }
+      }
+    )
+
+    val visibility = getVisibility(STYLE, name)
+    return BasicResourceItem(STYLE, name, visibility, configuration, resourceValue)
+  }
+
+  @Throws(IOException::class, XmlPullParserException::class)
+  private fun createStyleableItem(
+    name: String,
+    file: PathString,
+    configuration: RepositoryConfiguration
+  ): BasicResourceItem {
+    val namespaceResolver = parser.namespaceResolver
+    val resourceValue = StyleableResourceValueImpl(namespace, name, null, null)
+    forSubTags(
+      TAG_ATTR,
+      object : XmlTagVisitor {
+        override fun visitTag() {
+          val attrName = parser.getAttributeValue(null, ATTR_NAME)
+          if (attrName != null) {
+            try {
+              val item = createAttrItem(
+                attrName,
+                file,
+                configuration
+              )
+              val value = item.resourceValue as AttrResourceValue
+              resourceValue.addValue(value)
+
+              if (item.namespace == namespace && (namespace != ResourceNamespace.RES_AUTO || value.formats.isNotEmpty())) {
+                addValueResourceItem(item)
+              }
+            } catch (e: XmlSyntaxException) {
+              handleParsingError(file, e)
+            }
+          }
+        }
+      }
+    )
+
+    // AAPT2 treats all styleable resources as public.
+    // See https://android.googlesource.com/platform/frameworks/base/+/master/tools/aapt2/ResourceParser.cpp#1539
+    resourceValue.namespaceResolver = namespaceResolver
+    return BasicResourceItem(
+      STYLEABLE,
+      name,
+      ResourceVisibility.PUBLIC,
+      configuration,
+      resourceValue
+    )
+  }
+
+  /**
+   * Adds attr definitions from [.myAttrs], and attr definition candidates from [.myAttrCandidates]
+   * if they don't match the attr definitions present in [.myAttrs].
+   */
+  private fun processAttrsAndStyleables() {
+    // for (attr in myAttrs.values()) {
```

Comment:
ok for now since we're iterating on this quickly, but in general, deleted code >> commented-out code